content/tooling/api/bindings.md

```diff
@@ -40,13 +39,12 @@ curl -X GET "https://api.cloudflare.com/client/v4/zones/023e105f4ecef8ad9ca31a83
 
 ##### URL Parameters
 
-- `account_id`: the identifier associated with your Cloudflare account. [Find your Cloudflare Account ID](/quickstart#finding-your-cloudflare-api-keys)
+- `account_id`: the identifier associated with your Cloudflare account. [Find your Cloudflare Account ID](/quickstart/#configure)
 - `script_name`: the name of the script to be deleted.
 
-##### Headers ([Find Your Auth Info](/quickstart#finding-your-cloudflare-api-keys))
+##### Headers ([Find Your Auth Info](/quickstart/#configure))
```

Comment:
could the parenthetical part here be moved down into a paragraph? e.g.

```
##### Headers

Find your auth info
```

Reply:
probably! this is how it's been but I don't see why not.